pkg/kubectl/cmd/util/cmdproto/k8s_io_kubectl_cmd/cmd.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto2";
+package k8s.io.kubectl.cmd;
+
+import "google/protobuf/descriptor.proto";
+
+message FlagDetail {
+  required string name = 1;
+  repeated string value = 2;
+  required string shorthand = 3;
+  required string usage = 4;
+  enum EXT {
+      NONE = 0;
```

Comment:
Why are these necessary?

Reply:
the `enum EXT` is for some mapping variable between cobra and protobuf, for instance if I want to create a cobra `timeDuration` flag, I need to declare an Int64 in proto file and EXT = ISTIME.